### ISSUE TITLE
TSV filtered data download not working

### ISSUE THREAD

**Issue by mestato** [statonlab/Treesnap-website#374]
To recreate: Log in as admin, click on Admin menu item, click on Filters side pane. Click download icon and select TSV format. No file downloads, nothing happens.

Ditto for my Observations -> filters -> download -> tsv.

CSV works fine.

**Comment by almasaeed2010** [comment]
TSV works for me. I think if we have too many observations in a filter, it'll fail because it's taking too long to produce the file. This is a bigger issue that will require creating a queue to handle file creation or we can easily overload our little server.

**Comment by mestato** [comment]
Ah, I probably didn't wait long enough. Okay, I'll alter this to a feature request, ie testing to see how many observations creates an issue, and if needed, figuring out a queue to handle big file creation.

**Comment by almasaeed2010** [comment]
I added a few performance optimizations that might help the download file get created faster but this means we needed more ram so I chunked the data so it's possible to process in our small server. Load time for 800 observations went down from 400 seconds to 240 seconds so almost 50% faster.  

@mestato would be nice to know if your filter is downloadable.

This is more or less a hack to get this to work so we'll eventually need a queue system (and a UI) to handle download requests better.
